backend/src/apiserver/server/util.go

```diff
@@ -342,9 +311,9 @@ func isAuthorized(resourceManager *resource.ResourceManager, ctx context.Context
 	}
 
 	glog.Info("Getting user identity...")
-	userIdentity, err := getUserIdentity(ctx)
+	userIdentity, err := resourceManager.IsRequestAuthenticated(ctx)
```

Comment:
Yes definitely!

Reply:
Ah, we need to have it public since it's called by `isAuthorized()` of package `server` ([source](https://github.com/arrikto/kubeflow-pipelines/blob/ac4936fbbfb44ff98a5267026925d7cfed1645e8/backend/src/apiserver/server/util.go#L314)) (same as `IsRequestAuthorized()`).
So it needs to be `AuthenticateRequest` instead of `authenticateRequest`.